recipes/libgeotiff/all/patches/fix-cmake-1.6.0.patch

```diff
@@ -16,28 +18,7 @@
  
  SET(CMAKE_COLOR_MAKEFILE ON)
  
-@@ -54,7 +54,7 @@ IF(CMAKE_BUILD_TYPE MATCHES Debug)
-     SET(GEOTIFF_BUILD_PEDANTIC TRUE)
- ENDIF()
- 
--if (CMAKE_MAJOR_VERSION GREATER 2)
-+if(0) # No ! we want CMP0042 NEW for relocatable shared lib on macOS
- cmake_policy(SET CMP0022 OLD) # interface link libraries
- cmake_policy(SET CMP0042 OLD) # osx rpath
- endif()
-@@ -80,8 +80,9 @@ SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
- 
- IF(WIN32)
-     IF(MSVC)
--        ADD_DEFINITIONS(-DBUILD_AS_DLL=1)
--        ADD_DEFINITIONS(/DW4)
-+        IF(BUILD_SHARED_LIBS)
-+            ADD_DEFINITIONS(-DBUILD_AS_DLL=1)
-+        ENDIF()
```

Comment:
Please don't remove this, there is even a comment explaining why :/

Reply:
@SpaceIm 
Fixed.